server/utilities/neuralNetworkUtils.js

```diff
@@ -0,0 +1,37 @@
+const NN = require('../neural-network.js');
+const Product = require('../models/product.js');
+const User = require('../models/user.js');
+const synaptic = require("synaptic");
+
+const convertWinetoNeuron = (wine) => {
+  let type = wine.redORwhite === 'Red Wines' ? 1 : 0;
+  let price = wine.priceMin / 100;
+  let cab = wine.type === 'Cabernet Sauvignon' ? 1 : 0;
+  let merlot = wine.type === 'Merlot' ? 1 : 0;
+  let chard = wine.type === 'Chardonnay' ? 1 : 0;
+  let sauv = wine.type === 'Sauvignon Blanc' ? 1 : 0;
+  wine.neurons = [type, cab, merlot, chard, sauv, price];
+  return wine;
+}
+
```

Comment:
Not a big deal, I would move the let price = ... to line 13 so it is in the order that it appears in the array. Will do when I merge